beacon-chain/p2p/subnets.go

```diff
@@ -156,3 +261,11 @@ func (s *Service) subnetLocker(i uint64) *sync.RWMutex {
 	}
 	return l
 }
+
+func determineSize(bitCount int) int {
```

Comment:
looking for a better name, maybe `byteCount` and in comment mention it takes the ceiling